app/controllers/documents/prior_tax_returns_controller.rb

```diff
@@ -5,7 +5,7 @@ def self.show?(intake)
     end
 
     def self.document_type
-      "2018 Tax Return"
+      "Prior Year Tax Return"
```

Comment:
Changing document type strings may break things. We should either accompany this with a data migration or separate out the type key from the display name.

Reply:
In this case, it's an improvement, so I would lean towards a data migration.